configuration/pih/scripts/visit/encounterTypeConfig.js

```diff
@@ -255,7 +255,7 @@ angular.module("encounterTypeConfig", [])
             longTemplate: "templates/sections/viewSectionWithHtmlFormLong.page",
             templateModelUrl: "/htmlformentryui/htmlform/viewEncounterWithHtmlForm/getAsHtml.action?encounterId={{encounter.uuid}}&definitionUiResource=" + getFormResource("hiv/section-eid-protection.xml"),
             editUrl: "/htmlformentryui/htmlform/editHtmlFormWithStandardUi.page?patientId={{visit.patient.uuid}}&visitId={{visit.uuid}}&encounterId={{encounter.uuid}}&definitionUiResource=" + getFormResource("hiv/section-eid-protection.xml") + "&returnUrl={{returnUrl}}&breadcrumbOverride={{breadcrumbOverride}}",
-            require: "patientAgeInYearsOnDate(visit.startDatetime) < 2"
```

Comment:
@mogoodrich / @lnball - should this also be changed to 18 months as a part of this ticket, or is this supported to remain at 2 years?

Reply:
@mseaton I think it's possible that the exposed infant followup form could be data entered beyond 18 months.  It's not a question for us but of ZL...

Reply:
@lnball - this question is specifically about the "eidProtection" section on the follow-up form.  I'll leave it alone, just wanted to bring it to attention.